.github/workflows/deploy.yml

```
@@ -0,0 +1,43 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/configure-pages@v4
+      - name: Prepare artifact
+        run: |
+          mkdir -p public
+          cp index.html windsurf_logo_wordmark.png styles.css public/
+      - uses: actions/upload-pages-artifact@v3
+        with:
+          path: public
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)